package.json

```diff
@@ -48,11 +52,19 @@
     ]
   },
   "devDependencies": {
-    "ava": "^0.11.0",
-    "xo": "^0.12.1"
+    "ava": "^0.22.0",
+    "eslint": "^4.8.0",
+    "eslint-config-standard-with-typescript": "^1.0.1",
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-node": "^5.2.0",
+    "eslint-plugin-promise": "^3.5.0",
+    "eslint-plugin-standard": "^3.0.1",
+    "npm-run-all": "^4.1.1",
+    "typescript": "^2.5.3",
+    "typescript-eslint-parser": "^8.0.0"
   },
   "dependencies": {
     "is-plain-object": "^2.0.1",
-    "path-to-regexp": "^1.2.1"
+    "path-to-regexp": "^1.7.0"
```

Comment:
This version still works. The 2.0.0 does not, because it dropped support for the asterisk.